open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventProducer.java.orig

```diff
@@ -0,0 +1,548 @@
+/* SPDX-License-Identifier: Apache 2.0 */
```

Comment:
Do we need this file?  I would have expected that we would let Git manage the change history for us, rather than having `.orig` files scattered around (?)